docs/docs/2-core-concepts/2-blocks/1-your-first-block.mdx

```diff
@@ -492,6 +490,6 @@ Now you should be ready to create additional blocks! Maybe even a Dog Ear block?
 
 :::info
 
-We've only covered the basic concepts of COMET DXP blocks in this guide. To get a full picture, please review the [blocks](../Blocks/) section in our documentation.
+We've only covered the basic concepts of COMET DXP blocks in this guide. To get a full picture, please review the [blocks](/docs/core-concepts/blocks/) section in our documentation.
```

Comment:
Has this caused any issues as we upgraded to React 18? Seems somehow unrelated to me.

Reply:
> Note: This upgrades Docusaurus as well. After spending quite some time trying to fix issues between React 17 and React 18 types, I've decided that it's easier to upgrade Docusaurus in this PR. Sorry for the many changes 😬

This is caused by the Docusaurus upgrade.